torch_xla/experimental/custom_kernel.py

```diff
@@ -955,6 +955,28 @@ def ragged_paged_attention(
   # in the global scope which could cause problems for xmp.spawn.
   from torch_xla.experimental.pallas_kernels.ragged_paged_attention_v2 import ragged_paged_attention as ragged_attention
 
+  if num_kv_pages_per_block is None and num_queries_per_block is None:
+    token_num = q.shape[0]
+    # This heristic is based on the initial kernel micro benchmarking:
+    # When the token_num is small, there's no long request of prefill.
+    # While when it's larger, the block size is adjusted for it.
+    if token_num <= 128:
```

Comment:
I wonder if we should choose the block sizes in vLLM. If we choose to do in torch_xla, then we need to change it in torch_xla and wait for the wheel tmr. If we do in vLLM, it'd be more convenient. wdyt?

Reply:
If vLLM pass a non-None block size, the default value will not be used.

Reply:
I wonder why we couldn't do `_get_default_ragged_paged_attention_block_size` in vLLM..

Reply:
Usually it's a good idea to put the tuned-parameter table in the kernel lib, not in the app lib.